negotiator_directory_sync/auth/__init__.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+
+import requests
+
+from ..clients.negotiator_client import NegotiatorAPIClient
+from ..exceptions.TokenExpiredException import TokenExpiredException
+from ..logger import LOG
+
+AUTH_OIDC_TOKEN_URI = os.getenv('AUTH_OIDC_TOKEN_URI')
+AUTH_CLIENT_ID = os.getenv('AUTH_CLIENT_ID')
+AUTH_CLIENT_SECRET = os.getenv('AUTH_CLIENT_SECRET')
```

Comment:
same as above. Would it not be better tho parse and handle all ENVs in a separate file and then just import the constants?

Reply:
Fixed in [4df2a93](https://github.com/BBMRI-ERIC/directory-negotiator-sync/pull/1/commits/4df2a934ebed4a56a823da4d2fbaec1eb68ef804)